suave/e2e/suave-ex-node/keystore/UTC--2023-09-04T11-50-59.559521488Z--b5feafbdd752ad52afb7e1bd2e40432a485bbb7f

```diff
@@ -0,0 +1 @@
+{"address":"b5feafbdd752ad52afb7e1bd2e40432a485bbb7f","crypto":{"cipher":"aes-128-ctr","ciphertext":"8075ff2ed17c6cf6cd162b4bdd2926034e2067f03055990d57510e5d807ef06e","cipherparams":{"iv":"8c31b77d9518a68fda4aa6c90d62562d"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"6e55b1eea32430c4dc0b87cfc31168d552249b8ba946314e3c41dbeaeed3d125"},"mac":"5e411244fd732deb4464d247cfeb9beadc8a37558f12720c4d2ee8691436c50c"},"id":"51d12702-2276-44a9-972e-2011c56edf4e","version":3}
```

Comment:
This is not required once we merge https://github.com/flashbots/suave-geth/pull/161

Reply:
This is missing an newline at the end of the file, as well :)